src/huggingface_hub/hf_api.py

```diff
@@ -1082,8 +1112,8 @@ def create_repo(
                 "Ignoring provided space_sdk because repo_type is not 'space'."
             )
 
-        if lfsmultipartthresh is not None:
-            json["lfsmultipartthresh"] = lfsmultipartthresh
+        if getattr(self, "_lfsmultipartthresh", None):
+            json["lfsmultipartthresh"] = self._lfsmultipartthresh
```

Comment:
Why was this changed?

Reply:
The docstring of this parameter says:

    lfsmultipartthresh: Optional: internal param for testing purposes.

And it doesn't seem to affect the logic of the code.

It doesn't seem like a parameter which should be a part of the public API and exposed to the user, and it's a trivial fix in the tests to make it work this way. Happy to move it to a separate PR if you think it needs to be separate.